web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -71,17 +71,60 @@ Proposing a PDEP is done by creating a PR adding a new file to `web/pdeps/`.
 The file is a markdown file, you can use `web/pdeps/0001.md` as a reference
 for the expected format.
 
-The initial status of a PDEP will be `Status: Under discussion`. This will be changed to
-`Status: Accepted` when the PDEP is ready and has the approval of the core team.
+The initial status of a PDEP will be `Status: Draft`. This will be changed to
+`Status: Under discussion` by the author(s), when they are ready to proceed with the descision
+making process. 
+
```

Comment:
Could we have a subsection "How a PDEP becomes Accepted" (or different title), and then having schedule/voting/quorum be subsection of this one?